src/styles/pages/index.scss

```diff
@@ -389,14 +464,28 @@ $logos-zh: bank-of-china pingan guangda bank-of-beijing webank lufax ke-finance
         }
 
         width: 1344px;
-        height: 540px;
+        height: 534px;
         margin: 0 2rem;
         background-image: url(../../../images/home/architecture/zh/architecture.svg);
       }
     }
   }
 
   .section-learn-more {
+    h2 {
+      margin-bottom: 4rem !important;
+    }
+
+    .columns {
```

Comment:
There are some places using self-defined `.columns`, I would suggest to use mixin.